docs/enums/NeoVisEvents.html

```diff
@@ -16,7 +16,7 @@
 <li><a href="NeoVisEvents.html">NeoVisEvents</a></li></ul>
 <h1>Enumeration NeoVisEvents</h1></div><aside class="tsd-sources">
 <ul>
-<li>Defined in <a href="https://github.com/neo4j-contrib/neovis.js/blob/cef0aa1/src/events.ts#L3">src/events.ts:3</a></li></ul></aside>
+<li>Defined in <a href="https://github.com/neo4j-contrib/neovis.js/blob/5b175cb/src/events.ts#L3">src/events.ts:3</a></li></ul></aside>
```

Comment:
Can we define this blob id in constants why if it predefined and use that constant here in url

Reply:
This is for testing specific commits